### ISSUE TITLE
README page width not the same as on GitHub

### ISSUE THREAD

**Issue by joshbainbridge** [yusukebe/gh-markdown-preview#43]
Firstly, this is great extension, thanks for all the hard work 🙏

I recently noticed the Markdown body of any README page is rendering at 890px when using `markdown-preview`, as opposed to 830px when viewing the same page directly on GitHub.

I'm not sure if getting a one-to-one match is important to the project, but thought it worth mentioning in case it wasn't known. I'd presume the fix might be as simple as:

```html
--- a/cmd/template.html
+++ b/cmd/template.html
@@ -27,7 +27,7 @@
       .markdown-body {
         box-sizing: border-box;
         min-width: 200px;
-        max-width: 980px;
+        max-width: 920px;
         margin: 0 auto;
         padding: 45px;
       }
```

Tested on Safari Version 16.5 with macOS 13.4.

**Comment by yusukebe** [comment]
Hi @joshbainbridge 

Okay, let's fix it as you said. Could you create the PR for it? Thanks!

**Comment by joshbainbridge** [comment]
No problem! https://github.com/yusukebe/gh-markdown-preview/pull/44

**Comment by yusukebe** [comment]
@joshbainbridge 

Shipped new version `v1.4.1` includes your fix. Check it out.

**Comment by joshbainbridge** [comment]
@yusukebe awesome, thanks! Looks like it is now matching. I'll close the issue.
